os/pm/pm_metrics.c

```diff
@@ -23,6 +23,32 @@
 #include "pm_metrics.h"
 #include "pm.h"
 
+struct pm_metric_global_s g_pm_metrics_globals;
```

Comment:
The pm_metric should not be a static value as it is executed by the tash command. Please change it to allocate and free through actions.